experimental/codepointtrie/src/fast.rs

```diff
@@ -0,0 +1,143 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
question: Is any of this code still being used? Ignoring it for now since it doesn't seem to be.